### ISSUE TITLE
About the released model

### ISSUE THREAD

**Issue by numb1313** [JakeRenn/AI-Challenger-Keypoints-pytorch#2]
I tested validation set using your released pre-train model. But the result is just about 0.486, which is much worse than the result of testB. **Is this model the best model you used for submitting?** Or is it possible that I made something wrong during the test process.

**Comment by JakeRenn** [comment]
The mAP you get is correct.

You need some tricks to improve the performance, Such as multi-scale and embedding. Bigger output feature map also helps.

**Comment by Minotaur-CN** [comment]
What do you mean embedding? 
Could u reply a link for reference?

Thanks for your help!

**Comment by JakeRenn** [comment]
就是模型融合@Minotaur-CN

**Comment by c1a1o1** [comment]
How to install your nms？ @dang-qi  @Minotaur-CN
